script/hassfest/manifest.py

```diff
@@ -171,6 +174,20 @@ def verify_wildcard(value: str):
     return value
 
 
+def verify_lower_case_dict(*keys: Any) -> Callable[[dict], dict]:
```

Comment:
Can't you replace this with

```python
verify_lower_case_dict = vol.Schema({verify_lowercase: verify_lowercase})
```

it will verify all keys and values are lowercase.

Reply:
I think so

Reply:
Thats way better 👍